indico/modules/events/editing/client/js/components/FileManager/reducer.js

```diff
@@ -78,6 +78,18 @@ export default combineReducers({
     if (action.type === actions.RESET) {
       return action.fileTypes;
     }
+    if (action.type === actions.INVALID_FILENAME) {
+      return state.map(fileType => {
+        if (fileType.id === action.fileTypeId) {
+          return {
+            ...fileType,
+            invalidFiles: fileType.invalidFiles.filter(ift => ift !== action.filename),
```

Comment:
s/ift/name/ (earlier i wasn't sure what it was so i just abbreviated whatever long name there was)